templates/mongo-replicaset/0/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+mongo-init:
+  restart: 'no'
```

Comment:
Can this be made into a sidekick? Having a service completely stopped, makes the whole stack go into a degraded mode. If you only need to run it once, and it needs to be run by only one container, you can copy what we did for glusterfs.

[Elect a leader](https://github.com/rancher/compose-templates/blob/master/glusterfs/containers/0.1.3/glusterfs/lowest_idx.sh)
[Leader script](https://github.com/rancher/compose-templates/blob/master/glusterfs/containers/0.1.3/glusterfs/replicated_volume_create.sh#L22)